vcpkg/maintainers/functions/vcpkg_msbuild_install.md

```diff
@@ -0,0 +1,121 @@
+---
+title: vcpkg_msbuild_install
+description: Learn how to use vcpkg_msbuild_install.
+ms.date: 08/14/2023
+---
+
+# vcpkg_msbuild_install
+
+Build and install a msbuild-based project. This replaces `vcpkg_install_msbuild` and all other msbuild related functions.
+`vcpkg_msbuild_install` will generate a props and target file containing the cmake toolchain build flags and include it via `/p:ForceImportAfterCpp(Props|Targets)`
+
+## Usage
+
+```cmake
+vcpkg_install_msbuild(
+    SOURCE_PATH <${SOURCE_PATH}>
+    PROJECT_SUBPATH <port.sln>
+    [ADD_BIN_TO_PATH]
+    [NO_INSTALL]
+    [NO_TOOLCHAIN_PROPS]
+    [CLEAN]
+    [RELEASE_CONFIGURATION <Release>]
+    [DEBUG_CONFIGURATION <Debug>]
+    [TARGET <Build>]
+    [PLATFORM <Win32>]
+    [OPTIONS </p:ZLIB_INCLUDE_PATH=X>...]
+    [OPTIONS_RELEASE </p:ZLIB_LIB=X>...]
+    [OPTIONS_DEBUG </p:ZLIB_LIB=X>...]
+    [DEPENDENT_PKGCONFIG <zlib>]
+    [ADDITIONAL_LIBS <getopt.lib>]
+    [ADDITIONAL_LIBS_DEBUG <zlibd.lib>]
+    [ADDITIONAL_LIBS_RELEASE <zlib.lib>]
+)
+```
+
+## Parameters
+
+### SOURCE_PATH
+The path to the root of the source tree.
+
+Because MSBuild uses in-source builds, the source tree will be copied into a temporary location for the build. This
+parameter is the base for that copy and forms the base for all XYZ_SUBPATH options.
+
+
+### PROJECT_SUBPATH
+
+The subpath to the solution (`.sln`) or project (`.vcxproj`) file relative to `SOURCE_PATH`.
+
+### ADD_BIN_TO_PATH
+
+Adds the configration dependent binary path to the `PATH` environment variable before the build.
```

Comment:
```suggestion
Adds the configuration dependent binary path to the `PATH` environment variable before the build.
```

What is meant by "configuration dependent binary path"? We should be more specific if this means that it adds `installed/<triplet>/[debug/]bin`.

Reply:
https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/maintainers/functions/vcpkg_cmake_install.md#add_bin_to_path

is this more specific?